projects/ngx-charts-on-fhir/src/lib/fhir-data/fhir-data.service.spec.ts

```diff
@@ -3,13 +3,18 @@ import { FhirDataService } from './fhir-data.service';
 import FHIR from 'fhirclient';
 import { fhirclient } from 'fhirclient/lib/types';
 import { cold, getTestScheduler } from 'jasmine-marbles';
+import { ReportBPComponent } from 'projects/cardiovascular-patient-app/src/app/report-bp/report-bp.component';
+
+const bloodPressure = { systolic: 89, diastolic: 63 }
 
 describe('FhirDataService', () => {
   let service: FhirDataService;
 
   beforeEach(() => {
     sessionStorage.clear();
-    TestBed.configureTestingModule({});
+    TestBed.configureTestingModule({
+      providers: [{ provide: ReportBPComponent, useValue: bloodPressure }]
+    });
```

Comment:
Why is this providing an object constant for a component token? Does this actually work? Is anything in this test injecting this?

Reply:
I checked this and nothing in this test is injecting ReportBPComponent. This can be removed.